<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageOutput</class>
 <widget class="QWidget" name="pageOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Filename pattern:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default output directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditOutputDirectory"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditFilenamePattern"/>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelPreview">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonChooseDirectory">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>The following placeholders can be used for the filename pattern
and for the output directory. If used for the latter it is possible
to create a folder structure like ~/screenshots/${YYYY}.

${YYYY}    year, 4 digits
${MM}    month, 2 digits
${DD}    day, 2 digits
${hh}    hour, 2 digits
${mm}    minute, 2 digits
${ss}    second, 2 digits
${description}    e. g. window title; disallowed characters are replaced with _
${description_}    like ${description_} and all spaces are replaced with _
${user}    system user
${hostname}    machine/PC name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
